index.bs

```diff
@@ -1576,6 +1576,11 @@ partial dictionary MLOpSupportLimits {
     <td>[=/any data type|any=]</td>
     <td>[=/any rank|N=]</td>
   </tr>
+  <tr>
+    <td>*output*</td>
+    <td>{{MLArgMinMaxOptions/outputDataType}}</td>
+    <td>0 to {{input}}'s [=MLOperand/rank=]</td>
```

Comment:
```suggestion
    <td>{{input}}'s [=MLOperand/rank=] - 1 to {{input}}'s [=MLOperand/rank=]</td>
```
The output must be either the same rank as the input or one less (if keepDimensions == false).